packages/node_modules/@ciscospark/plugin-calendar/src/index.js

```diff
@@ -36,6 +36,16 @@ registerPlugin(`calendar`, Calendar, {
         extract(response) {
           return Promise.resolve(response.body);
         }
+      },
+      {
+        name: `transformMeetingFromMercury`,
```

Comment:
This should be achievable without a new transformation. this predicate should be also be named `tranformMeeting`, but should provide a different `extract()` than the `transformMeeting` above.